tool/docker/base/Dockerfile_postgres_11_Jenkins

```diff
@@ -16,25 +16,26 @@
 # specific language governing permissions and limitations
 # under the License.
 
-FROM postgres:11
+FROM postgres:11-bullseye
 
 ### Get postgres specific add-ons
-RUN apt-get update && apt-get install -y  wget \
+RUN apt-get update
+RUN apt-get install -y  wget \
                        build-essential \
                        postgresql-server-dev-11 \
-                       postgresql-plpython-11 \
+                       postgresql-plpython3-11 \
                        openssl \
                        libssl-dev \
                        libboost-all-dev \
                        m4 \
                        rpm \
-                       python-pip \
-                       python-dev \
+                       python3-pip \
+                       python3-dev \
                        build-essential \
                        cmake \
                        libspatialindex-dev
 
-RUN pip install tensorflow==1.14 dill "rtree>=0.8,<0.9"
+RUN python3 -m pip install tensorflow dill rtree xgboost mock pandas numpy scikit-learn
```

Comment:
don't we need the specific versions anymore?

Reply:
With python3 the latest versions work as expected. If we start to get any issues, we should get an error and try to fix it, or at least pin it to a different version.